addons/mail/static/src/models/composer/composer.js

```diff
@@ -566,6 +566,32 @@ function factory(dependencies) {
             return [['unlink', unmentionedChannels]];
         }
 
+        /**
+         * Executes the given async function, only when the last function
+         * executed by this method terminates. If there is already a pending
+         * function it is replaced by the new one. This ensures the result of
+         * these function come in the same order as the call order, and it also
+         * allows to skip obsolete intermediate calls.
+         *
+         * @param {function} func
+         */
+        async _executeOrQueueFunction(func) {
+            if (this._hasMentionRpcInProgress) {
+                this._nextMentionRpcFunction = func;
+                return;
+            }
+            this._hasMentionRpcInProgress = true;
+            this._nextMentionRpcFunction = undefined;
+            try {
+                await func();
+            } finally {
+                this._hasMentionRpcInProgress = false;
+                if (this._nextMentionRpcFunction) {
+                    this._executeOrQueueFunction(this._nextMentionRpcFunction);
+                }
+            }
+        }
```

Comment:
missing `this.async()`

Reply:
It probably doesn't matter at all in this case, especially with the  `finally`. But I guess it doesn't hurt to add it.

What might be better to add though is to actually check for exists before executing the function